### ISSUE TITLE
Oprávnění ze skautíku

### ISSUE THREAD

**Issue by malekjakub69** [skaut/Skautske-hospodareni#2830]
### Popiš chybu
Nemohu jako admin okresu, ani jako admin oddílu spravovat cizí cestovní příkazy a už několik let nám tam visí neuzavřené cestovní příkazy, kterých bych se ráz zbavil, tím že je uzavřu. 

### Jak chybu nasimulovat
Kroky pro nasimulování chyby:
1. Otevři h.skauting.cz
2. Přejdi na stránku cestovní příkazy 
3. Zkuso otevřít cestovní příkaz který jsi nevytvořil (s rolí admina oddílu nebo okresu)
4. Neoprávněný přístup

### Očekávané chování
Možnost editovat i cizí cestovní příkazy, pokud mám roli admina 

### Snímky obrazovky
![image](https://github.com/user-attachments/assets/423a5c3d-37e2-4dfa-b01e-b739b806cb7d)

### Uživatelské informace:
 - Sekce: Cestovní příkazy
 - Uživatelská role: vedoucí/admin střediska 614.10.006 / admin okres 614
 - Cestovní příkaz https://h.skauting.cz/cestaky/default/detail/2074

**Comment by juradee** [comment]
Díky za report, podívám se na to

**Comment by sinacek** [comment]
Cestak https://h.skauting.cz/cestaky/default/detail/2074 je vázaný na jednotku `Jednotka 614.10 - přístav Žlutá Ponorka Třebíč` takze kdyz budes mit aktivni roli `vedouci/admin 614.10`, tak by ti to mělo mit zavrit

**Comment by malekjakub69** [comment]
Ano, to jsem si musel zažádat, abych mohl vůbec s cizími cesťáky manipulovat. Nešlo by tedy udělat, že cesťáky, které nemohu otevřít, tak se mi vůbec nezobrazí a nebo alespoň zobrazit autora cesťáku, abych věděl na koho se obrátit, aby cesťák uzavřel ?  

A ještě k nadřazeným rolím: Myslel jsem si, že pokud mám nadřazenou roli, tedy admin - okres, tak moh dělat vše co mohou admin role v podřízených jednotkách, což se tu ukázalo, že ne.
